docs/pages/application-access/okta/guide.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Setting up the Okta service
+description: How to set up the Okta service with some basic configurations.
+---
+
+This guide will help you set up the Okta service and provide some basic
```

Comment:
Similarly, _why_ might I want to provide basic configurations to the Okta service? What can I actually accomplish by doing so?

Reply:
@mdwn I agree with Zac here, let's give a few bullet points of what you can actually achieve by setting up Okta integration: import apps/groups, use access requests to provide temporary access, etc.